### ISSUE TITLE
Error: Cannot find module '~/env' Require stack: - /qw/src/constants.ts

### ISSUE THREAD

**Issue by chillpilllike** [vendure-ecommerce/storefront-qwik-starter#166]
I am getting this error when I start the storefront:
Could you please help

> start
> pnpm generate && vite --open --mode ssr

> storefront-qwik-starter@ generate /qw
> pnpm generate-shop && pnpm generate-admin

> storefront-qwik-starter@ generate-shop /qw
> graphql-codegen --config codegen-shop.ts

Error: Cannot find module '~/env'
Require stack:
- /qw/src/constants.ts
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at _resolve (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:241814)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:244531)
    at /qw/src/constants.ts:2:12
    at evalModule (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at /qw/codegen-shop.ts:2:18
    at evalModule (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at loadTypeScript (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/helpers/cosmiconfig.js:38:12)
    at #loadConfiguration (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:92:30)
    at #readConfiguration (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:77:80)
    at async load (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:20:48)
    at async Explorer.load (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:23:20)
    at async getConfig (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/helpers/get-config.js:10:17)
    at async loadConfig (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/config.js:30:15)
    at async findAndLoadGraphQLConfig (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/graphql-config.js:35:20)
    at async loadContext (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/config.js:79:27)
    at async createContext (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/config.js:190:21)
    at async runCli (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/cli.js:37:21) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/qw/src/constants.ts' ],
  filepath: '/qw/codegen-shop.ts'
}
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.

> start
> pnpm generate && vite --open --mode ssr

> storefront-qwik-starter@ generate /qw
> pnpm generate-shop && pnpm generate-admin

> storefront-qwik-starter@ generate-shop /qw
> graphql-codegen --config codegen-shop.ts

Error: Cannot find module '~/env'
Require stack:
- /qw/src/constants.ts
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at _resolve (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:241814)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:244531)
    at /qw/src/constants.ts:2:12
    at evalModule (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at /qw/codegen-shop.ts:2:18
    at evalModule (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/qw/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at loadTypeScript (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/helpers/cosmiconfig.js:38:12)
    at #loadConfiguration (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:92:30)
    at #readConfiguration (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:77:80)
    at async load (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:20:48)
    at async Explorer.load (/qw/node_modules/.pnpm/cosmiconfig@8.3.6_typescript@5.3.3/node_modules/cosmiconfig/dist/Explorer.js:23:20)
    at async getConfig (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/helpers/get-config.js:10:17)
    at async loadConfig (/qw/node_modules/.pnpm/graphql-config@5.1.0_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/graphql-config/cjs/config.js:30:15)
    at async findAndLoadGraphQLConfig (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/graphql-config.js:35:20)
    at async loadContext (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/config.js:79:27)
    at async createContext (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/config.js:190:21)
    at async runCli (/qw/node_modules/.pnpm/@graphql-codegen+cli@5.0.2_@types+node@20.11.17_graphql@16.9.0_typescript@5.3.3/node_modules/@graphql-codegen/cli/cjs/cli.js:37:21) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/qw/src/constants.ts' ],
  filepath: '/qw/codegen-shop.ts'
}
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.

**Comment by gioboa** [comment]
Hi, thanks for your feedback.
Let me try with a fresh clone.

**Comment by chillpilllike** [comment]
Just checking, if you were able to reproduce the error or need more information.

**Comment by gioboa** [comment]
It's working fine :thinking: 
```
git clone git@github.com:vendure-ecommerce/storefront-qwik-starter.git
cd storefront-qwik-starter.git
pnpm install
pnpm dev
```

**Comment by chillpilllike** [comment]
I am trying to run a production build using a dockerfile on server

FROM node:20
WORKDIR /qw
RUN apt-get update && apt-get install -y git && apt-get clean
RUN npm install -g pnpm
RUN git clone https://github.com/vendure-ecommerce/storefront-qwik-starter.git .
RUN pnpm install
CMD ["pnpm", "start"]

and this error comes

**Comment by gioboa** [comment]
I fixed the error, thanks for your feedback @chillpilllike it helps the project :smile:

**Comment by chillpilllike** [comment]
Hello, thank, after this update the env error is resolved but this error shows up:

[STARTED] Parse Configuration
[SUCCESS] Parse Configuration
[STARTED] Generate outputs
[STARTED] Generate to src/generated/graphql-admin.ts
[STARTED] Generate to src/generated/schema-admin.graphql
[STARTED] Load GraphQL schemas
[STARTED] Load GraphQL schemas
[SUCCESS] Load GraphQL schemas
[SUCCESS] Load GraphQL schemas
[STARTED] Load GraphQL documents
[STARTED] Load GraphQL documents
[SUCCESS] Load GraphQL documents
[SUCCESS] Load GraphQL documents
[STARTED] Generate
[STARTED] Generate
[FAILED] GraphQL Document Validation failed with 1 errors;
[FAILED]   Error 0: Cannot query field "orders" on type "Query". Did you mean "order"?
[FAILED]     at /qw/src/providers/admin/order.ts:3:3
[FAILED] GraphQL Document Validation failed with 1 errors;
[FAILED]   Error 0: Cannot query field "orders" on type "Query". Did you mean "order"?
[FAILED]     at /qw/src/providers/admin/order.ts:3:3
[SUCCESS] Generate outputs
 ELIFECYCLE  Command failed with exit code 1.

**Comment by gioboa** [comment]
[Fixed](767be2323cdeef33e2f2195a11172dbfbeb73058)
